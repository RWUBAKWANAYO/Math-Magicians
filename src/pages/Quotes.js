import React from 'react';
import { MdFormatQuote } from 'react-icons/md';

const Quotes = () => (
  <div className="quotes-cont">
    <div>
      <MdFormatQuote className="left-quote" />
      <p>
        Mathematics is not about numbers, equations, computations, or
        algorithms: it is about understanding. -William Paul Thurston
      </p>
      <MdFormatQuote className="right-quote" />
    </div>
  </div>
);

export default Quotes;
