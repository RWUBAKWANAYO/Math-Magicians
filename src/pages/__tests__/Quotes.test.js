import renderer from 'react-test-renderer';
import Quotes from '../Quotes';

it('Quotes page render', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
