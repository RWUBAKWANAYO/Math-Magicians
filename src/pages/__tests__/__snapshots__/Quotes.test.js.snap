// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes page render 1`] = `
<div
  className="quotes-cont"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
